daemon/daemon_unix.go

```diff
@@ -112,10 +112,11 @@ func getCPUResources(config containertypes.Resources) *specs.CPU {
 	}
 
 	if config.NanoCPUs > 0 {
-		// Use the default setting of 100ms, as is specified in:
+		// We set the highest value possible (1s), as is specified in:
 		// https://www.kernel.org/doc/Documentation/scheduler/sched-bwc.txt
-		//    	cpu.cfs_period_us=100ms
-		period := uint64(100 * time.Millisecond / time.Microsecond)
+		//    	cpu.cfs_period_us=1s
+		// The purpose is to get the highest precision
+		period := uint64(1 * time.Second / time.Microsecond)
```

Comment:
This change looks much more correct to me, I think its originally wrong in swarm for the calculation